mylyn.builds/org.eclipse.mylyn.builds.ui/META-INF/MANIFEST.MF

```diff
@@ -57,4 +56,5 @@ Export-Package: org.eclipse.mylyn.builds.ui;x-internal:=true,
  org.eclipse.mylyn.internal.builds.ui.notifications;x-internal:=true,
  org.eclipse.mylyn.internal.builds.ui.util;x-internal:=true,
  org.eclipse.mylyn.internal.builds.ui.view;x-internal:=true
+Import-Package: org.osgi.service.event;version="1.4.1"
```

Comment:
In general it is good practice to specify an upper bound to follow semantic versioning.
```suggestion
Import-Package: org.osgi.service.event;version="[1.4.0,2.0.0)"
```

Reply:
Might be good improvement for PDE :)

Reply:
We are working on it :)
https://github.com/eclipse-pde/eclipse.pde/pull/1007#pullrequestreview-1782918647

Reply:
Great! I have improved the constraints according to your recommendations.